### DISCUSSION TITLE
Error Installing Brew

### DISCUSSION THREAD

**Discussion by braddalton** [Homebrew/discussions#5542] [Getting started]
### Output of `brew config`

```shell
Not installed
```

### Output of `brew doctor`

```shell
Not installed
```

### Description of issue

==> This script will install:
/usr/local/bin/brew
/usr/local/share/doc/homebrew
/usr/local/share/man/man1/brew.1
/usr/local/share/zsh/site-functions/_brew
/usr/local/etc/bash_completion.d/brew
/usr/local/Homebrew

Press RETURN/ENTER to continue or any other key to abort:
==> /usr/bin/sudo /usr/sbin/chown -R braddalton:admin /usr/local/Homebrew
==> Downloading and installing Homebrew...
remote: Enumerating objects: 276744, done.
remote: Counting objects: 100% (1564/1564), done.
remote: Compressing objects: 100% (836/836), done.
error: RPC failed; curl 92 HTTP/2 stream 5 was not closed cleanly: CANCEL (err 8)
error: 2312 bytes of body are still expected
fetch-pack: unexpected disconnect while reading sideband packet
fatal: early EOF
fatal: fetch-pack: invalid index-pack output
Failed during: /usr/bin/git fetch --force origin
braddalton@Brads-iMac ~ %

**Comment by SMillerDev** [comment]
See #666
